packages/flutter_tools/lib/src/base/clock.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:quiver/time.dart';
+
+import 'context.dart';
```

Comment:
I'd probably also restrict the usage of quiver to the actual tests and just actually pass in a callback to get the current time but that's even less of a real problem.